docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -17,6 +17,15 @@ this `blog post <http://pulpproject.org/2016/11/17/django14-epel6-retirement/>`_
 motivation for this decision.
 
 
+Pulp 2.12.2
```

Comment:
This should probably become it's own PR so that when it's cherry picked into an existing GA release these release notes won't come with it.